static/css/impala/header.less

```diff
@@ -41,6 +41,7 @@
 .site-title img {
     float: left;
     margin-right: 10px;
+    width: 62px;
```

Comment:
Should this be in the main style? I thought everything went into the one `restyle.less`.